backend/index.js

```diff
@@ -117,7 +117,7 @@ app.use((req, res, next) => {
 app.use(async (req, res, next) => {
   res.locals = {
     ...res.locals,
-    currentUrlPath: req.originalUrl,
+    currentUrlPath: req.baseUrl + req.path,
```

Comment:
Couldn't find anywhere that uses query string parameters so didn't add an int test